src/components/UI/NavigationBar/NavigationBar.tsx

```diff
@@ -1,63 +1,78 @@
-import ChevronLeftIcon from '@mui/icons-material/ChevronLeft'
 import MenuIcon from '@mui/icons-material/Menu'
 import { Divider, Drawer, IconButton, Toolbar, Typography } from '@mui/material'
 import { MenuList } from 'components/UI/NavigationBar/MenuList/MenuList'
 import { AppBar } from 'components/UI/NavigationBar/AppBar'
 import { DrawerHeader } from 'components/UI/NavigationBar/DrawerHeader'
 import { NavigationBarContextProvider } from 'components/UI/NavigationBar/NavigationBarContext'
 import * as React from 'react'
+import CacheService from "../../../services/cache_service";
```

Comment:
Eventually everything that's in the services directory will be removed. The preferred pattern is to use react hooks directly. In the case of browser cache, the replacement is `useLocalStorage` and the preferred way to import it is:

```typescript
import { useLocalStorage } from 'hooks/useLocalStorage'
```

i.e., without using a relative path. The API of the hook is fairly straightforward and the code that needs the functionality should need only minor changes.